vllm/platforms/rocm.py

```diff
@@ -129,15 +129,25 @@ def check_and_update_config(cls, vllm_config: VllmConfig) -> None:
         scheduler_config = vllm_config.scheduler_config
         if parallel_config.worker_cls == "auto":
             if scheduler_config.is_multi_step:
-                parallel_config.worker_cls = \
-                    "vllm.worker.multi_step_worker.MultiStepWorker"
+                if envs.VLLM_USE_V1:
+                    raise NotImplementedError
```

Comment:
From `vllm/platforms.cuda.py`:
```suggestion
                    raise NotImplementedError(
                          "Multi-step scheduling is not supported (and not "
                          "needed) on VLLM V1. Please launch without "
                          "--num-scheduler-steps.")
```